pkg/minikube/driver/driver.go

```diff
@@ -47,7 +47,7 @@ const (
 	HyperKit = "hyperkit"
 	// VMware driver
 	VMware = "vmware"
-	// VMwareFusion driver
+	// VMwareFusion driver (obsolete)
```

Comment:
You could also just delete it ? That is what we did with kvm and xhyve, for instance (0332f7335)